### ISSUE TITLE
Any finetune guide for the latest moondream3 ?

### ISSUE THREAD

**Issue by nendonerd** [vikhyat/moondream#310]
Any finetune guide for the latest moondream3 ?

**Comment by Milan933-coder** [comment]
Using JEPA .Joint Embedding PRedictive Architecture is the way where the model is trained with an embedding model .Like the language head wuill be separated and then the output is converted through a bigger model and then the input is passed also then the loss is compared
